### ISSUE TITLE
Bot commented saying PR was closed unmerged when it wasn't

### ISSUE THREAD

**Issue by edmorley** [renovateapp/renovate#1205]
Hi - thank you for a great tool!

The renovate bot just commented on a PR with the "As this PR has been closed unmerged, Renovate will ignore this upgrade ..." message:
https://github.com/mozilla-neutrino/neutrino-dev/pull/401#issuecomment-345926689

...however the PR had in fact been merged after all (from the commit message I'm guessing the GitHub "squash and merge" feature was used).

Is this expected?

Many thanks :-)

**Comment by rarkins** [comment]
Thanks for the report. I've figured out what happened. Renovate had the PR opened updating 24 package files. Around the time it was merged, https://github.com/mozilla-neutrino/neutrino-dev/commit/39599a410d250148172f82047e7b1929a1c6ed21 was also merged, which added in another 1.x deepmerge which then was missed out. Race condition etc etc. 

Renovate tries to be smart and not recreate any of its PRs where you already had the same branch name/title before and closed it. Eg if you had closed the PR instead of merging it as you wanted to stick to 1.x.

Renovate needs to be a little smarter and not block "identical" PRs if the first one was *merged* and not closed. I will fix this soon, but in the meantime you can unblock the new PR by renaming the existing PR in any way at all (eg add a ! to the end).

**Comment by edmorley** [comment]
Ah makes sense - thank you for taking a look. I can see this being a problem space where there is a significant amount of complexity around the edges! :-)
